ChangeLog.md

```diff
@@ -20,6 +20,10 @@ See docs/process.md for more on how version tagging works.
 
 3.1.57 (in development)
 -----------------------
+- JS library code is now executed in a limited context/scope.  Only certain
+  symbols are made available in this scope.  If there are build time JS symbols
+  that you are depending on, but that were not added to this scope, please
+  file a bug and we can get them added. (#21542)
```

Comment:
Maybe another way to put it is: "JS library code is now executed in a limited context/scope, which limits how much of the compiler internals you can access. [..]" - ? Just an idea